openbao.yaml

```diff
@@ -46,11 +50,9 @@ subpackages:
     description: "Compatibility package to allow replacing vault image"
     pipeline:
       - runs: |
-          mkdir -p "${{targets.subpkgdir}}/bin"
-          ln -sf /usr/bin/bao "${{targets.subpkgdir}}/bin/bao"
-          ln -sf /bin/bao "${{targets.subpkgdir}}/bin/vault"
-
           mkdir -p "${{targets.subpkgdir}}/usr/bin"
+          ln -sf /usr/bin/bao "${{targets.subpkgdir}}/usr/bin/vault"
```

Comment:
Relative would be better. But also inconsequential.

Reply:
Agreed- fixed.